src/KafkaW/Producer.cpp

```diff
@@ -164,15 +164,15 @@ Producer::Producer(Producer &&x) noexcept {
 }
 
 void Producer::poll() {
-  int events_handled =
+  int EventsHandled =
       rd_kafka_poll(RdKafkaPtr, ProducerBrokerSettings.PollTimeoutMS);
   LOG(Sev::Debug,
       "IID: {}  broker: {}  rd_kafka_poll()  served: {}  outq_len: {}", id,
-      ProducerBrokerSettings.Address, events_handled, outputQueueLength());
+      ProducerBrokerSettings.Address, EventsHandled, outputQueueLength());
   if (log_level >= 8) {
```

Comment:
thinking about it, we should just remove this. the log level should never be higher than 7 anyway.